tests/runtests/python-addon/runtest.sh

```diff
@@ -57,6 +59,37 @@ rlJournalStart
         rlRun "abrt-cli info $crash_PATH | grep $TFILE" 0 "abrt-cli info should contain $TFILE"
         rlRun "abrt-cli info $crash_PATH | grep 'Python'" 0 "abrt-cli info should contain 'Python'"
 
+        rlRun "abrt-cli rm $crash_PATH"
+    rlPhaseEnd
+
+    rlPhaseStartTest "RequireAbsolutePath test"
+        AASPD_CONF="/etc/abrt/abrt-action-save-package-data.conf"
+        PYTHON_CONF="/etc/abrt/plugins/python.conf"
+        rlFileBackup $AASPD_CONF $PYTHON_CONF
+
+        CRASH_PY="crash.py"
+        cat > $CRASH_PY << EOF
+#!/usr/bin/python
+1/0
+EOF
+        rlRun "chmod +x $CRASH_PY" 0
+
+        rlRun "augtool set /files${AASPD_CONF}/ProcessUnpackaged yes" 0
+        rlRun "augtool set /files${PYTHON_CONF}/RequireAbsolutePath yes" 0
+
+        prepare
+        rlRun "./$CRASH_PY" 1
+        rlAssertNotExists $crash_PATH
```

Comment:
This asserts test if `$crash_PATH` does not exist but if the crash does not exist, `$crash_PATH` is empty. What's worse, `$crash_PATH` is populated by the previous test which also removes the related problem, so the variable is not empty and the path will never exist.